doc/aws.md

```diff
@@ -0,0 +1,29 @@
+coreos-assembler and AWS
+===
+
+Using `cosa buildextend-aws` looks for AWS credentials in the standard places;
+Common methods are `AWS_ACCESS_KEY_ID`
+and `AWS_SECRET_ACCESS_KEY` environment variables, and using `AWS_PROFILE`.
+For more information, consult the documentation for the `aws` CLI tool, in particular
+`aws configure help`.
+
+Uploading AMIs [requires IAM permissions](https://docs.aws.amazon.com/vm-import/latest/userguide/vmie_prereqs.html#iam-permissions-image).
+
+Follow the [instructions to create the vmimport role](https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-image-import.html) and attach it to the bot account + bucket:
```

Comment:
Might be worth noting one can also use `ore aws initialize` to do this.

Reply:
OK tweaked to mention that.